apps/api/src/db/schemas/actions.ts

```diff
@@ -24,7 +24,7 @@ import { revisionRequests } from './revisionRequests.ts';
 // TODO: Integrate w/ TS
 // import { applicationActions } from 'pcgl-daco/packages/data-model/';
 
-const actionsEnum = pgEnum('actions', [
+export const actionTypesEnum = pgEnum('action_types', [
```

Comment:
Exporting enums due to bug described here: https://github.com/drizzle-team/drizzle-orm/issues/646#issuecomment-1586349095

Also updated to use lower snake case naming conventions; add `types` to distinguish btwn enums + tables